### DISCUSSION TITLE
Streamcontroller to control smart home

### DISCUSSION THREAD

**Discussion by Totte23** [StreamController/StreamController#260] [Show and tell]
Really great project!!!
I am running Streamcontroller on an OrangePi zero 3 with Armbian and xfce to control part of my smart home (iobroker) via RESTAPI.
I just have a mini deck but enough for a few pages to control lights, shutters, sockets, ...
To have display of returned value is great to show status of the different objects directly on the button and with your icons database it is looking really great.

Thanks a lot to all that are supporting this project!

**Comment by Core447** [comment]
Thank you!
